src/index.html

```diff
@@ -1,17 +1,168 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form method="post" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="fieldset">
+        <legend class="legend">Personal Information</legend>
+        <label class="field-wrap"> Surname:
+          <input
+            class="input"
+            type="text"
+            name="surName"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="field-wrap"> Name:
```

Comment:
it would be better to add empty line between siblings for better readability
```suggestion
        </label>
        
        <label class="field-wrap"> Name:
```

Reply:
check other places as well